src/models/BonusNumber.js

```diff
@@ -0,0 +1,58 @@
+import LOTTO_CONSTANTS from '../constants/lottoConstatns.js';
+import ERROR_MESSAGES from '../constants/messages/errorMessages.js';
+
+class BounsNumber {
+  #winningNumbers;
+  #bonusNumber;
+
+  constructor(winningNumbers, bonusNumber) {
+    this.#winningNumbers = winningNumbers;
+    this.#validateBonusNumber(bonusNumber);
+    this.#bonusNumber = Number(bonusNumber);
+  }
```

Comment:
생성자에서 `Number`로 보너스 넘버를 형 변환을 해줌에도 해당 속성을 클래스 내부에서 사용하지 않고 있는거로 보여요

아래와 같이 구현하지 않으신 이유가 있으실까요?

```suggestion
  constructor(winningNumbers, bonusNumber) {
    this.#winningNumbers = winningNumbers;
    this.#bonusNumber = Number(bonusNumber);
    this.#validateBonusNumber(this.#bonusNumber);
  }
```